features/cellular/framework/API/CellularDevice.h

```diff
@@ -37,6 +42,22 @@ namespace mbed {
  */
 class CellularDevice {
 public:
+
+    /** Return singleton instance of CellularDevice.
+     *
+     */
+    static CellularDevice *get_default_instance();
+
+    /** Get event queue that can be chained to main event queue. EventQueue is created in get_default_instance() or
+     *  given to CELLULAR_DEVICE (for example TELIT_HE910 class).
+     *  @return event queue
+     */
+    virtual events::EventQueue *get_queue();
```

Comment:
const method? Although might be a bit of an odd-one-out.

Reply:
Added const, maybe it will start to spread and no more odd-one-out 👍